scripts/gendocs/generate-new-import-path-docs

```diff
@@ -116,13 +117,6 @@ genclireference() {
     return
   fi
 
-  # hacky way to figure out if the target has the correct structure
-  # differs between older version but this catches the corner cases
-  if [[ ! -d "$2/app" ]] && [ ! -d "$2/cmd" ]; then
-    echo "+++ app directory for $2 does not exist, skipping..."
-    return
-  fi
-
```

Comment:
I don't understand what this check was for and it doesn't work with startupapicheck, which has neither of these sub-directories, so I removed it.